<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexView</class>
 <widget class="QMainWindow" name="IndexView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1102</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1102</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>myRin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/MainIcons.qrc">
    <normaloff>:/icon/mainIcon/Icons/calendar3.svg</normaloff>:/icon/mainIcon/Icons/calendar3.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>231</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="redaction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>526</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Редактирование</string>
     </attribute>
     <widget class="QPushButton" name="AddXmlButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>353</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить из 
XML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DownloadXmlButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>211</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить в XML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UploadJsonButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>211</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить из JSON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DownloadJsonButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>211</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить в JSON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddPerson">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeletePerson">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EditPerson">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить запись</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UploadXmlButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>353</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить из 
XML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CleanTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Очисть таблицу и базу данных</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="filters">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>526</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Фильтры</string>
     </attribute>
     <widget class="QLineEdit" name="FirstNameInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="SurnameInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PatronymicInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EmailInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TelephoneInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TelegramInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>@</string>
      </property>
     </widget>
     <widget class="QLabel" name="FirstNameLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
     <widget class="QLabel" name="SurnameLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
     <widget class="QLabel" name="PatronymicLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Отчество</string>
      </property>
     </widget>
     <widget class="QLabel" name="EmailLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="TelephoneLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
     <widget class="QLabel" name="TelegramLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FindButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CleanButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>490</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сбросить</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="DescriptionInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DescriptionLable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дополнительные сведения</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="overview">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>526</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Обзор</string>
     </attribute>
     <widget class="QLineEdit" name="TelephoneView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PatronymicView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="EmailViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SurnameView">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="FirstNameViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
     <widget class="QLabel" name="PatronymicViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Отчество</string>
      </property>
     </widget>
     <widget class="QLabel" name="SurnameViewLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="DescriptionView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EmailView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DescriptionViewLable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дополнительные сведения</string>
      </property>
     </widget>
     <widget class="QLabel" name="TelegramViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram</string>
      </property>
     </widget>
     <widget class="QLabel" name="TelephoneViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FirstNameView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="TelegramView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="CityTelephoneViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Городской телефон</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CityTelephoneView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="InternalTelephoneViewLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Внутренний телефон</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InternalTelephoneView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="PersonTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>811</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>16</width>
      <height>641</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Resources/MainIcons.qrc"/>
 </resources>
 <connections/>
</ui>
